src/main/resources/templates/search.html

```diff
@@ -13,7 +13,7 @@
 <body class="center">
 	<div class="item-wrapper">
 		<h1><span>商品情報検索</span></h1>
-		<form th:action="@{/id-search}" th:object="${itemSearchRequest}" th:method="post">
+		<form th:action="@{/item/${'id'}}" th:object="${itemSearchRequest}" th:method="get">
```

Comment:
以下のようにしてみるのはどうですか？
item → itemsにしているのでControllerの修正もお願いします！

```suggestion
		<form th:action="@{/items" th:object="${itemSearchRequest}" th:method="get">
```